src/OrleansAWSUtils/Options/DynamoDBMembershipTableOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrleansAWSUtils.Configuration
+{
+    public class DynamoDBMembershipTableOptions
+    {
+        /// <summary>
+        /// Deployment Id.
+        /// </summary>
+        public string DeploymentId { get; set; }
+        /// <summary>
+        /// Connection string for DynamoDB Storage
+        /// </summary>
+        public string DataConnectionString { get; set; }
```

Comment:
naming: ConnectionsString, I see no need for Data, since there is only 1 connection string for this provider, no need to differentiate.